config/locales/helpers.en.yml

```
@@ -133,7 +133,9 @@ en:
           over_20_months: More than 20 months
       english_language_provider:
         name: Name
-        b2_level_requirement: B2 level requirement
+        b2_level_requirement: B2 level requirement score
+        b2_level_requirement_prefix: B2 level requirement prefix
+        url: Provider url
```

```suggestion
        url: Website
```

This is probably clearer than URL for most users of the support console.